config/application.rb

```diff
@@ -253,7 +253,7 @@ def cache_store_config
     email_sanitizer_configs = (three_scale.delete(:email_sanitizer) || {}).symbolize_keys
     config.three_scale.email_sanitizer.merge!(email_sanitizer_configs)
 
-    config.three_scale.merge!(three_scale.slice!(:force_ssl, :access_code))
+    config.three_scale.merge!(three_scale.slice!(:force_ssl))
```

Comment:
I think there was actually a bug here. This `access_code` is retrieved here: https://github.com/3scale/porta/blob/3f78b0f6d496c47513f51d5a9722cae9134070e3/lib/developer_portal/lib/access_code_protection.rb#L42

But with this code, it was resulting in `nil`. The correct way to get the value would be `Rails.application.config.access_code`. But I thought it would be better to place it under `three_scale` specific config.